README.md

```diff
@@ -0,0 +1,94 @@
+# @hmcts/cmc-draft-store-middleware
+
+[![Travis badge](https://api.travis-ci.org/hmcts/cmc-draft-store-middleware.svg?branch=master)](https://travis-ci.org/hmcts/cmc-draft-store-middleware)
+
+[![Codecov badge](https://codecov.io/github/hmcts/cmc-draft-store-middleware/coverage.svg?branch=master)](https://codecov.io/github/hmcts/cmc-draft-store-middleware?branch=master)
+
+[![NPM version badge](https://img.shields.io/npm/v/@hmcts/cmc-draft-store-middleware.svg)](https://www.npmjs.com/@hmcts/cmc-draft-store-middleware)
+
+[![Node version badge](https://img.shields.io/node/v/@hmcts/cmc-draft-store-middleware.svg)](https://www.npmjs.com/@hmcts/cmc-draft-store-middleware)
+
+[![Greenkeeper badge](https://badges.greenkeeper.io/hmcts/cmc-draft-store-middleware.svg)](https://greenkeeper.io/)
+
+[![Standard - JavaScript Style Guide](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com/)
+
+This is a Node.js/Express library for interacting with the [draft store API](https://github.com/hmcts/draft-store).
+The middleware retrieves draft of selected type from the draft store API and sets it in Express.js local user scope.
+If draft does not exist in the draft store then an empty draft is set it in Express.js local user scope.
+
+## Getting started
+
+### How to use it?
+
+#### Installation
+
+To add library to the project dependencies run:
+
+```
+$ yarn add @hmcts/cmc-draft-store-middleware
+```
+
+or
+
+```
+$ npm install @hmcts/cmc-draft-store-middleware
+```
+
+#### Sample code
+
+Example use of the middleware in the Express.js application is presented below:
+
+```
+import { DraftMiddleware } from '@hmcts/cmc-draft-store-middleware/dist/middleware/draftMiddleware'
```

Comment:
It seems my comment got lost somewhere. Please investigate a way of making this import statement simpler. It should work as every typical Node module, i.e.:
```
import { DraftMiddleware } from '@hmcts/cmc-draft-store-middleware'
```
I believe this is a matter is providing a proper `index.js` file in module's root.

Reply:
Most likely, just not something we used to do for other module. Wanted to do it as next step to speed things up as story I am working on in not even within the sprint, some silent left over from last one :(